src/JS/main.js

```diff
@@ -1,3 +1,44 @@
 import { mobileMenu } from "./mobilmenu.js";
+import { getMovies } from "./getMovies.js";
 
 mobileMenu();
+
+(async function () {
+  let moviesArray = await getMovies();
+
+  moviesArray
+    .filter((movie) => movie.category === "Komedi")
+    .slice(0, 3)
+    .forEach((item) => {
+      let movieItem = document.createElement("li");
+      let movieImg = document.createElement("img");
+      movieImg.src = item.image[0];
+      movieItem.classList.add("movies-list-item");
+      document.querySelector(".comedy-start-list").appendChild(movieItem);
+      movieItem.appendChild(movieImg);
+    });
+
+  moviesArray
+    .filter((movie) => movie.category === "Action")
+    .slice(0, 3)
+    .forEach((item) => {
+      let movieItem = document.createElement("li");
+      let movieImg = document.createElement("img");
+      movieImg.src = item.image[0];
+      movieItem.classList.add("movies-list-item");
+      document.querySelector(".action-start-list").appendChild(movieItem);
+      movieItem.appendChild(movieImg);
+    });
+
+  moviesArray
+    .filter((movie) => movie.category === "Drama")
+    .slice(0, 3)
+    .forEach((item) => {
+      let movieItem = document.createElement("li");
+      let movieImg = document.createElement("img");
+      movieImg.src = item.image[0];
+      movieItem.classList.add("movies-list-item");
+      document.querySelector(".drama-start-list").appendChild(movieItem);
+      movieItem.appendChild(movieImg);
+    });
+})();
```

Comment:
This code throws error on all pages except startpage where it runs. Maybe fix that so it only runs code if user are on startpage/ index.html.

Reply:
Fix is on place. Now when it will be merged it will show only on start page movies och ska inte försöka rendera dem på andra sidor.